README.md

```diff
@@ -118,6 +118,32 @@ To run the integration tests of the PG connector using wal2json, enable the "wal
 A few tests currently don't pass when using the wal2json plug-in.
 Look for references to the types defined in `io.debezium.connector.postgresql.DecoderDifferences` to find these tests.
 
+### Building the Oracle connector
+
+In order to build the Debezium Oracle connector, the following prerequisites must be met:
+
+* Oracle DB is installed and enabled for change data capturing (TODO: describe the details)
+* The Instant Client is downloaded (e.g. [from here](http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html) for Linux) and unpacked
```

Comment:
Which of the packages? Basic/basic light etc.?

Reply:
I used Basic, though I reckon Basic Light works, too.